README.md

```diff
@@ -60,10 +68,8 @@ Screenshots & Gifs
 
 <kbd><img title="Visualizations" src="https://raw.githubusercontent.com/apache/incubator-superset/master/superset-frontend/images/screenshots/visualizations.png"></kbd><br/>
 
-Apache Superset
+## Why Superset
 ---------------
```

Comment:
Let's get rid of the `----------------` line here which makes a redundant `<hr/>`

Reply:
Done! :)